.evergreen/tox.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -o errexit  # Exit the script with error if any of the commands fail
+
+PYTHON_BINARY="$1"
+
+. .evergreen/utils.sh
+
+if $PYTHON_BINARY -m tox --version; then
+    run_tox() {
+      $PYTHON_BINARY -m tox "$@"
+    }
+elif python3 -m tox --version; then
+    run_tox() {
+      python3 -m tox "$@"
+    }
+elif tox --version; then
```

Comment:
I just realized that we can't use `python3 -m tox` or `tox --version` because those would end up using an incorrect python version (it wouldn't use $PYTHON_BINARY). Let's remove the two elifs.